.github/workflows/compilation_on_android_ubuntu.yml

```diff
@@ -380,6 +383,25 @@ jobs:
             ./core/deps/llvm/build/share
           key: ${{ matrix.llvm_cache_key }}
 
+      # - name: configure ubsan env
```

Comment:
probably best to delete it and add when actually needed; otherwise we might end up with a big block of commented code that might soon get out of sync if not maintained.

Reply:
yeah

I'm planning to close this PR and reopen for UBSAN only